linphone.xcodeproj/project.pbxproj

```diff
@@ -2487,11 +2491,14 @@
 				220FAD3910765B400068D98F /* libspeexdsp.a in Frameworks */,
 				226183AE1472527D0037138E /* libsrtp.a in Frameworks */,
 				22276E8513C73D3100210156 /* libswscale.a in Frameworks */,
+				63D7215D1B7394D200D70E65 /* libtunnel.a in Frameworks */,
```

Comment:
Should there be a tunnel.a here? Mostly concerned about introducing a private dependency